### DISCUSSION TITLE
Helm deployment fails on istio/gateway (with missing /proc/sys/net/ipv4/ip_unprivileged_port_start) but is working using istioctl operator (fortunately)

### DISCUSSION THREAD

**Discussion by sta4152** [istio/istio#49020] [General]
We are (yet) running our dev cluster on an old centos7 (3.10 kernel) linux. And struggled with deploying istio (latest, 1.20.2) following the docs for using helm on this. Ending up in not getting the chart `istio/gateway` deployed successfully as this found `/proc/sys/net/ipv4/ip_unprivileged_port_start` missing on the `proxyv2` container start:

`kubelet Failed to create pod sandbox: rpc error: code = Unknown desc = failed to create containerd task: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: error during container init: open /proc/sys/net/ipv4/ip_unprivileged_port_start: no such file or directory: unknown`

Just by accident I tested the same (? - at least I intended to) deployment using the `istioctl install` way. And to my surprise this is working, `proxyv2` container is up and running, ingress is working.

Any idea on the different result?
What does istioctl configure differently than when using helm?
And could one achieve the same using helm then, too?

Unfortunately I did not find the originating call causing this error within the sources so far.

**Comment by howardjohn** [comment]
Istioctl installs `manifests/charts/gateways/istio-ingress/`, helm `manifests/charts/gateway`. The latter is the "modern" version.

**Comment by Babbili** [comment]
i always use `istioctl` for installation, minimal profile, and configuring `IstioOperator` for any later modification to avoid similar issues
